src/NuGet.Clients/NuGet.Tools/NuGetUIFactory.cs

```diff
@@ -61,7 +61,8 @@ internal sealed class NuGetUIFactory : INuGetUIFactory
         public NuGetUIFactory(
             ICommonOperations commonOperations,
             INuGetUILogger logger,
-            ISourceControlManagerProvider sourceControlManagerProvider)
+            ISourceControlManagerProvider sourceControlManagerProvider,
```

Comment:
skip this from your PR. Pato will take care of that

Reply:
I have merged Pato's PR, will rebase from dev.